examples/examples/jsonrpsee_as_service.rs

```diff
@@ -58,30 +57,33 @@ struct Metrics {
 	http_connections: Arc<AtomicUsize>,
 }
 
+#[derive(Clone)]
 struct AuthorizationMiddleware<S> {
 	headers: HeaderMap,
 	inner: S,
 	#[allow(unused)]
 	transport_label: &'static str,
 }
 
-#[async_trait]
 impl<'a, S> RpcServiceT<'a> for AuthorizationMiddleware<S>
 where
 	S: Send + Sync + RpcServiceT<'a>,
```

Comment:
Should we also add the `Clone` bound to `S`? 🤔

Reply:
fixed, ty